pkg/virt-handler/cgroup/cgroup.go

```diff
@@ -84,13 +92,23 @@ func NewManagerFromPid(pid int) (manager Manager, err error) {
 	if runc_cgroups.IsCgroup2UnifiedMode() {
 		version = V2
 		slicePath := filepath.Join(cgroupBasePath, controllerPaths[""])
+		s := strings.Split(slicePath, "/")
+		if s[len(s)-1] == "housekeeping" {
+			fstr := strings.Join(s[1:len(s)-1], "/") + "/"
+			slicePath = fstr
+		}
 		manager, err = newV2Manager(config, slicePath)
 	} else {
 		version = V1
 		for subsystem, path := range controllerPaths {
 			if path == "" {
 				continue
 			}
+			s := strings.Split(path, "/")
```

Comment:
what do you think extracting this to a function? it seems to be duplicated.

Reply:
Fixed